CHANGELOG.md

```diff
@@ -8,6 +8,55 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 
 _No documentation available about unreleased changes as of yet._
 
+## [1.2.0] - 2018-11-12
+
+### Added
+- New `WordPress.PHP.TypeCasts` sniff to the `WordPress-Core` ruleset.
+    This new sniff checks that PHP type casts are:
+    * lowercase;
+    * short form, i.e. `(bool)` not `(boolean)`;
+    * normalized, i.e. `(float)` not `(real)`;
+   Additionally, the new sniff discourages the use of the `(unset)` and `(binary)` type casts.
+- New `WordPress.Utils.I18nTextDomainFixer` sniff which can compehensively replace/add `text-domain`s in a plugin or theme.
```

Comment:
Change: 
`text-domain`s
to:
text domains

Reply:
In this case, it's within `code` tags, which is why I let it mirror what you'd expect to find in code.